templates/batch-llm/README.md

```diff
@@ -128,18 +135,32 @@ model_name_to_input_prompt_format = {
     "mistralai/Mistral-7B-Instruct-v0.1": "[INST] {} [/INST]",
     "google/gemma-7b-it": "<start_of_turn>model\n{}<end_of_turn>\n",
     "mlabonne/NeuralHermes-2.5-Mistral-7B": "<|im_start|>system\nYou are a helpful assistant that will complete the sentence in the given input prompt.<|im_end|>\n<|im_start|>user{}<|im_end|>\n<|im_start|>assistant",
+    "meta-llama/Meta-Llama-3-8B-Instruct": (
+        "<|start_header_id|>system<|end_header_id|>\n\nYou are a helpful assistant. Complete the given prompt in several concise sentences<|eot_id|>\n"
+        "<|start_header_id|>user<|end_header_id|>\n\n{}<|eot_id|>\n"
+        "<|start_header_id|>assistant<|end_header_id|>\n\n"
+    ),
 }
 
 def construct_input_prompt(row, text_column):
-    if "[INST]" not in row[text_column]:
-        prompt_format = model_name_to_input_prompt_format.get(HF_MODEL)
-        if prompt_format:
-            row[text_column] = prompt_format.format(row[text_column])
+    """Given the input row with raw text in `text_column` column,
+    construct the input prompt for the model."""
+    prompt_format = model_name_to_input_prompt_format.get(HF_MODEL)
+    if prompt_format:
+        row[text_column] = prompt_format.format(row[text_column])
     return row
 
 ds = ds.map(construct_input_prompt, fn_kwargs={"text_column": "item"})
 ```
 
+So far, we have defined two operations of the Dataset (`from_items()`, `map()`), but have not executed the Dataset yet and don't see any results. Why is that?
```

Comment:
moved this up from previous section. since this is the first point where we call `map()` without executing the dataset, user may be confused why there is nothing running.